creator-node/test/issueSyncRequest.jobProcessor.test.js

```diff
@@ -439,4 +457,57 @@ describe('test issueSyncRequest job processor', function () {
     )
     expect(recordSuccessStub).to.have.not.been.called
   })
+
+  describe.only('test SYNC_MODES.MergePrimaryAndSecondary', async function () {
+    syncMode = SYNC_MODES.MergePrimaryAndSecondary
+
+    it.only('Issues correct sync when primarySyncFromSecondary() succeeds and no additional sync is required', async function () {
+      const getNewOrExistingSyncReqStub = sandbox.stub().callsFake((args) => {
+        throw new Error('getNewOrExistingSyncReq was not expected to be called')
+      })
+  
+      const getSecondaryUserSyncFailureCountForTodayStub = sandbox
+        .stub()
+        .returns(0)
+  
+      const retrieveClockValueForUserFromReplicaStub = sandbox.stub().resolves(1)
+
+      const primarySyncFromSecondaryStub = sandbox.stub().returns(null)
+  
+      const issueSyncRequestJobProcessor = getJobProcessorStub({
+        getNewOrExistingSyncReqStub,
+        getSecondaryUserSyncFailureCountForTodayStub,
+        retrieveClockValueForUserFromReplicaStub,
+        primarySyncFromSecondaryStub
+      })
+  
+      // Make the axios request succeed
+      nock(baseURL).post('/sync', data).reply(200)
+  
+      // Verify job outputs the correct results: no sync issued (nock will error if the wrong network req was made)
+      const result = await issueSyncRequestJobProcessor({
+        logger,
+        syncType,
+        syncMode,
+        syncRequestParameters
+      })
+      expect(result).to.have.deep.property('error', {})
+      expect(result).to.have.deep.property('jobsToEnqueue', {})
+      expect(result.metricsToRecord).to.have.lengthOf(1)
+      expect(result.metricsToRecord[0]).to.have.deep.property(
+        'metricName',
+        'audius_cn_issue_sync_request_monitoring_duration_seconds'
+      )
+      expect(result.metricsToRecord[0]).to.have.deep.property('metricLabels', {
+        syncType: 'manual',
+        reason_for_additional_sync: 'none'
+      })
+      expect(result.metricsToRecord[0]).to.have.deep.property(
+        'metricType',
+        'HISTOGRAM'
```

Comment:
this might've been copied from an older version -- it should now be 'HISTOGRAM_OBSERVE' (or `MetricRecordType.HISTOGRAM_OBSERVE` from `prometheus.constants`)

Reply:
ouch, bad merge - thx